iCubTelemVizServer/.idea/runConfigurations/iCubTelemServerStart.xml

```diff
@@ -0,0 +1,14 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="iCubTelemServerStart" type="js.build_tools.npm" singleton="false">
+    <package-json value="$PROJECT_DIR$/package.json" />
+    <command value="run" />
+    <scripts>
+      <script value="start" />
+    </scripts>
+    <node-interpreter value="project" />
+    <envs>
+      <env name="NODE_DEBUG_OPTION" value="--debug" />
+    </envs>
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
Just a double check, are you sure that all this `.idea` files are meant to be committed?

Reply:
Well, a priori no, but I wanted to save the debug configuration as it was a quicker and easier way to have a default debug configuration without having to extensiely document the procedure or going through all the documented use cases for anyone trying to debug the code. Then, in Webstorm, we just have to import the debug configruations instead of recreating them (in the best case where we remember how to do it).

(We can always remove them in future PRs.)

Reply:
Ok then!